workshop/content/40-dotnet/20-create-project/300-structure.md

```diff
@@ -0,0 +1,98 @@
++++
+title = "Project structure"
+weight = 300
++++
+
+## Open your IDE
+
+Now's a good time to open the project in your favorite IDE and explore.
+
+> If you use VSCode, you can just type `code .` within the project directory.
+
+## Explore your project directory
+
+You'll see something like this:
+
+![](./structure.png)
+
+//TODO
+* `src/CdkWorkshop/Program.cs` is the entrypoint for the CDK application it will load the stack defined in `src/CdkWorkshop/CdkWorkshopStack.cs`
+* `src/CdkWorkshop/CdkWorkshopStack.cs` is where your CDK application's main stack is defined. This is the file we'll be spending most of our time in.
+* `cdk.json` tells the toolkit how to run your app. In our case it will be
+  `"node bin/cdk-workshop.js"`
+* `src/CdkWorkshop/CdkWorkshop.csproj` is the C# project file. It is an xml file and contains information on references. This will be useful to you down the line, but is not relevant for the purposes of this workshop.
+* `src/CdkWorkshop.sln` is the C# solution file that provides build information. You should not need to interface with this file.
+* `.gitignore` and `.npmignore` tell git and npm which files to include/exclude
+  from source control and when publishing this module to the package manager.
+* The `src/CdkWorkshop/bin` and `src/CdkWorkshop/obj` folders are the build folders for the project and can be ignored.
+
+## Your app's entry point
+
+Let's have a quick look at `src/CdkWorkshop/Program.cs`:
+
+```c#
+using Amazon.CDK;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace CdkWorkshop
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            var app = new App(null);
```

Comment:
Remove the unneeded `null` arguments everywhere :P (You may want to re-generate the samples from the fixed up init templates once those have merged :D)